guidelines/sc/22/focus-appearance.html

```diff
@@ -7,11 +7,11 @@ <h4>Focus Appearance</h4>
     
     <p>When a <a>user interface component</a> has keyboard focus, one or both of the following is true:</p>
     <ol>
-        <li>The <a>focus indicator</a>:
+        <li>The entire <a>focus indicator</a>:
```

Comment:
```suggestion
        <li>The entire <a>focus indicator</a> meets all the following:
```

Reply:
I think "meets all the following" is redundant given "the entire" in front of it.Not something I particularly care about (in other words, if you feel strongly, I can live with it), but I just don't like adding more words that add little value.